cloud.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2020 The DDSP Authors.
```

Comment:
These files part of DDSP, if you are importing them to this repository, better to add them in a different directory `third_party/ddsp` and make it clear they come from another source. However to see which are your contributions and preserve the history I would suggest doing the PR against ddsp repo directly.